server/jamovi/server/formatio/omv.py

```diff
@@ -313,6 +317,7 @@ def read(data, path, prog_cb, **kwargs):
 
         meta_content = zip.read('metadata.json').decode('utf-8')
         metadata = json.loads(meta_content)
+        meta_document = metadata.get('document', None)
```

Comment:
you could do a:

`meta_document = metadata.get('document', {'resultsLanguage': 'en'})`

here if you wanted.

Reply:
... allows you to tidy up some subsequent code